.gitignore

```diff
@@ -1,5 +1,6 @@
 build
 .kdev4
+.vscode
```

Comment:
Is this necessary?

Reply:
Minor convenience - I do not use kdev4, but I use vs code.